terragrunt/log_archive/legacy_archives/main.tf

```diff
@@ -0,0 +1,11 @@
+module "aws-landing-zone-logs_bucket" {
+  source = "github.com/cds-snc/terraform-modules?ref=v3.0.19//S3_log_bucket"
+
+  billing_tag_value = var.billing_code
+}
+
+module "aws-landing-zone-s3-access-logs_bucket" {
+  source = "github.com/cds-snc/terraform-modules?ref=v3.0.19//S3_log_bucket"
+
+  billing_tag_value = var.billing_code
+}
```

Comment:
These buckets are specifically used to store access logs for other s3 buckets.

These should be basic S3 buckets: https://github.com/cds-snc/terraform-modules/tree/main/S3

Reply:
Fixed, changed to basic S3 module